In [25]:
import pandas as pd


df = pd.read_csv("new/xqcow.csv")
print(df)
df.drop(columns=['channel_id', 'commenter_id', 'commenter_type', 'fragments', 'offset', 'video_id', 'updated_at','Unnamed: 0'], inplace=True, axis=1)

df

     Unnamed: 0                   body  channel_id  commenter_id  \
0        396407                LUL LUL    71092938     150523129   
1       1037720        VESTON OMEGALUL    71092938      93592799   
2       1112379   LOUDER MrDestructoid    71092938     226840177   
3       1976987                    LUL    71092938     172840299   
4       2156709                   LULW    71092938      89522849   
..          ...                    ...         ...           ...   
245      387468                     D:    71092938     207437221   
246     2571198            xqc is cute    71092938     198744529   
247     1559214                POGGERS    71092938     147241083   
248     1125444                WutFace    71092938      65975401   
249     1694502  dafranPepo dafranPepo    71092938      62358340   

    commenter_type                created_at  \
0             user  2018-06-21T19:16:48.862Z   
1             user  2018-06-07T21:26:32.338Z   
2             user  2018-06-07T06:57:14

body                created_at
0                  LUL LUL  2018-06-21T19:16:48.862Z
1          VESTON OMEGALUL  2018-06-07T21:26:32.338Z
2     LOUDER MrDestructoid  2018-06-07T06:57:14.905Z
3                      LUL  2018-05-30T02:32:33.232Z
4                     LULW   2018-05-24T04:47:04.43Z
..                     ...                       ...
245                     D:  2018-06-21T18:36:03.538Z
246            xqc is cute  2018-05-20T01:44:02.302Z
247                POGGERS  2018-06-03T07:21:29.896Z
248                WutFace  2018-06-05T23:58:02.826Z
249  dafranPepo dafranPepo  2018-06-01T03:40:51.633Z

[250 rows x 2 columns]

Now that we have gotten the random 250 Messages, we will now parse and score them.

In [26]:
for message in df.body:
    print(message)

LUL LUL
VESTON OMEGALUL
LOUDER MrDestructoid
LUL
LULW
CoolStoryBob CoolStoryBob CoolStoryBob CoolStoryBob CoolStoryBob CoolStoryBob CoolStoryBob
11115511512BILLIONn
BANNED LULW BANNED LULW
DOOMFIST
WHAT HAPPENED TO MOXY BibleThump
OMEGALUL WTF
WutFace
ALLO ZULUL
If you buy battle net balance is that blizzard balance just another saying or wut..
KappaPride KappaPride KappaPride KappaPride KappaPride KappaPride KappaPride KappaPride
SHOW US THE JUICE
Ice DansGame
@CoolStorySara xqcL
M PogChamp X Y
BANNED AGAIN OMEGALUL
xomeryTalonwidow xomeryTalonwidow xomeryTalonwidow
C LUL C
AYAYA Weeb test. If you touch the AYAYA, you're a weeb ⎝ VoHiYo
WE HOPE U ENJOYED UR STAY xqcSad ITS GOOD TO HAVE U WITH US xqcSad EVEN IF ITS JUST FOR THE DAY xqcSad WE HOPE U ENJOYED UR STAY xqcSad ITS GOOD TO HAVE U WITH US xqcSad EVEN IF ITS JUST FOR THE DAY xqcSad WE
<3
random Sombra lol
xqcHug
Pog hygene
Do you lose more sr then you gain In masters?
@olivertheprice adeptthebest
POLICE xqcG
xqcRage MOXYS TURNx